packages/http/src/forwarder/index.ts

```diff
@@ -6,6 +6,7 @@ import * as NEA from 'fp-ts/NonEmptyArray';
 import * as E from 'fp-ts/Either';
 import * as TE from 'fp-ts/TaskEither';
 import * as RTE from 'fp-ts/ReaderTaskEither';
+import * as JSON from 'fp-ts/json';
```

Comment:
You shouldn't use `JSON` keyword, it is reserved by JavaScript. Better is to use `import * as J from 'fp-ts/Json'` instead.